Microsoft.Toolkit.Uwp.UI/Extensions/FrameworkElement/FrameworkElementExtensions.RelativeAncestor.cs

```diff
@@ -27,14 +27,14 @@ public static object GetAncestor(DependencyObject obj)
         /// <param name="value">FrameworkElement</param>
         public static void SetAncestor(DependencyObject obj, object value)
         {
-            obj.SetValue(AncestorProperty, value);
+            obj.SetValue(AncestorProperty, new WeakReference<object>(value));
         }
 
         /// <summary>
         /// Attached <see cref="DependencyProperty"/> for retrieving a parent <see cref="object"/> for the <see cref="AncestorProperty"/>
         /// </summary>
         public static readonly DependencyProperty AncestorProperty =
-            DependencyProperty.RegisterAttached("Ancestor", typeof(object), typeof(FrameworkElementExtensions), new PropertyMetadata(null));
+            DependencyProperty.RegisterAttached("Ancestor", typeof(WeakReference<object>), typeof(FrameworkElementExtensions), new PropertyMetadata(null));
```

Comment:
By changing the type here we break using it in XAML as intended. This can be demonstrated with the `TokenizingTextBox` as it uses this feature to grab the `TokenSpacing` property from its parent.